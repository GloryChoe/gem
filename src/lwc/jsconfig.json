{
    "compilerOptions": {
        "experimentalDecorators": true,
        "baseUrl": ".",
        "paths": {
            "c/sge_defaultGau": [
                "sge_defaultGau/sge_defaultGau.js"
            ],
            "c/sge_dynamicForm": [
                "sge_dynamicForm/sge_dynamicForm.js"
            ],
            "c/sge_formField": [
                "sge_formField/sge_formField.js"
            ],
            "c/sge_formSection": [
                "sge_formSection/sge_formSection.js"
            ],
            "c/sge_service": [
                "sge_service/sge_service.js"
            ]
        }
    },
    "include": [
        "**/*",
        "../../.sfdx/typings/lwc/**/*.d.ts"
    ],
    "typeAcquisition": {
        "include": [
            "jest"
        ]
    }
}
