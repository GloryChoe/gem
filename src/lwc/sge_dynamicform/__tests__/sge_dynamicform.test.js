import { createElement } from 'lwc';
import DynamicForm from 'c/sge_dynamicform';

describe('c-dynamicform', () => {
   it('should load', () => {
       const element = createElement('c-sge-dynamicform', { is: DynamicForm });
       document.body.appendChild(element);
       const div = element.shadowRoot.querySelector('lightning-accordion');
       console.log(document.body);
   });
});