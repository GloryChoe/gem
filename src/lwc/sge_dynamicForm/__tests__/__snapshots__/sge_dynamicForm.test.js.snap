// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`c-dynamicform getValues should work when values are blank or empty 1`] = `
Object {
  "npsp__In_Kind_Type__c": undefined,
  "npsp__Ask_Date__c": undefined,
  "npsp__Fair_Market_Value__c": undefined,
}
`;
